requirements-dev/regenerate.sh

```diff
@@ -5,7 +5,10 @@
 cd "${0%/*}"
 
 ./py37/regenerate.sh
+# ./py38/regenerate.sh
```

Comment:
delete commented out lines (and maybe add comment about symlink)

Reply:
Done in c22ac6e